packages/javascript/src/index.js

```diff
@@ -10,4 +10,4 @@ export * from "./gen/livekit_room_pb.js";
 export * from "./gen/livekit_rtc_pb.js";
 export * from "./gen/livekit_sip_pb.js";
 export * from "./gen/livekit_webhook_pb.js";
-export * from "./gen/version.js";
+export * from "./gen/version.mjs";
```

Comment:
why would we mix .js and .mjs imports here?

Reply:
the order of operations for the build is genversion → js (as CJS) → rename to cjs → js (as ESM).
if we genversion as `js` and not `mjs`, it'll get translated to cjs and overwrite the other one.
another approach would be to run genversion&js (CJS) → rename → genversion&js (ESM), but i feel like that's worse

Reply:
I see, but there's a separate `index.cjs` file, what's the relation to that then?

Reply:
that's because the only files getting renamed from js to cjs are inside gen.
basically:

- `genversion` (creates version.{m,c}js, that won't be renamed)
- `protoc` generates CJS files
- we rename all of those newly-generated files to `.cjs` from `.js` (this is the part where we can't genversion as js, because it'll override the cjs file)
- `protoc` as ESM.